src/quick-start.md

```diff
@@ -58,23 +72,23 @@ spec:
 
 Overview of the attributes of the `PolicyServer` resource:
 
-* `image`: container image name
-* `replicas`: number of desired instances
-* `serviceAccountName` (optional): `policy-server` ServiceAccount. Namespace's `default` ServiceAccount will be used if nothing is provided
-* `env` (optional): `policy-server` environment variables
-* `annotations` (optional): `policy-server` annotations
+| Required | Placeholder         | Description    |
+|:--------:| ------------------- | ----------------------------- |
+| ✅ | `image`  | The name of the container image |
+| ✅ | `replicas`  | The number of desired instances |
+| - | `serviceAccountName` | The name of the `ServiceAccount` to use for the `PolicyServer` deployment. If no value is provided, the default `ServiceAccount` from the namespace, where the `kubewarden-controller` is installed, will be used |
+| - | `env` | The list of environment variables |
+| - | `annotations` | The list of annotations |
 
-Changing any of these attributes will lead to a rollout of the `policy-server` Deployment with the new configuration.
+Changing any of these attributes will lead to a rollout of the `PolicyServer` Deployment with the new configuration.
 
-## Kubewarden Policies
+### ClusterAdmissionPolicy
 
-Enforcing policies is by far the most common operation a Kubernetes
-administrator will perform. You can declare as many policies as you want,
-targeting any kind of Kubernetes resource and type of operation that can be
-done against them.
+The `ClusterAdmissionPolicy` resource is the core of the Kubewarden stack. This resource defines how the policies are evaluated.
 
-The `ClusterAdmissionPolicy` resource is the core of the Kubewarden stack: this is
-how validating policies are defined.
+Enforcing policies is the most common operation which a Kubernetes administrator will perform. You can declare as many policies as you want, and each policy will target a specific Kubernetes resource (i.e. ` pods`) and list the type of operation that will be applied for the targeted resource. The operations available are `CREATE`, `UPDATE`, `DELETE` and `CONNECT`.
```

Comment:
While I think this is good for the quickstart, in principle there is no restriction of a policy having to target a single specific resource. It can target as many different types of resources it wants, and they don't have to be core Kubernetes types.

A policy might act on specific Custom Resources as well, and a single policy can target e.g. two different types of resources. Although it's probably not recommended, it's perfectly possible and up to the policy author.